src/front/domain_socket.rs

```diff
@@ -31,7 +31,7 @@ static SOCKET_PATH: &str = "/tmp/security-daemon-socket";
 ///
 /// Only works on Unix systems.
 pub struct DomainSocketListener {
-    listener: Option<UnixListener>,
```

Comment:
Makes sense to remove that to me! As the `DomainSocketListener` can only be instanciated by the builder, this will never be `None`.